x-pack/plugins/ml/public/application/trained_models/models_management/start_deployment_setup.tsx

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiForm,
+  EuiButtonGroup,
+  EuiFormRow,
+  EuiFieldNumber,
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiButtonEmpty,
+  EuiButton,
+  EuiCallOut,
+  EuiSpacer,
+  EuiDescribedFormGroup,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+} from '@elastic/eui';
+import { toMountPoint, wrapWithTheme } from '@kbn/kibana-react-plugin/public';
+import type { Observable } from 'rxjs';
+import type { CoreTheme, OverlayStart } from '@kbn/core/public';
+import { isCloud } from '../../services/ml_server_info';
+import {
+  composeValidators,
+  numberValidator,
+  requiredValidator,
+} from '../../../../common/util/validators';
+
+interface StartDeploymentSetup {
+  config: ThreadingParams;
+  onConfigChange: (config: ThreadingParams) => void;
+}
+
+export interface ThreadingParams {
+  numOfAllocations: number;
+  threadsPerAllocations: number;
+}
+
+const THREADS_MAX_EXPONENT = 6;
+
+/**
+ * Form for setting threading params.
+ */
+export const StartDeploymentSetup: FC<StartDeploymentSetup> = ({ config, onConfigChange }) => {
+  const numOfAllocation = config.numOfAllocations;
+  const threadsPerAllocations = config.threadsPerAllocations;
+
+  const threadsPerAllocationsOptions = useMemo(
+    () =>
+      new Array(THREADS_MAX_EXPONENT).fill(null).map((v, i) => {
+        const value = Math.pow(2, i);
+        const id = value.toString();
+
+        return {
+          id,
+          label: id,
+          value,
+        };
+      }),
+    []
+  );
+
+  const toggleIdSelected = threadsPerAllocationsOptions.find(
+    (v) => v.value === threadsPerAllocations
+  )!.id;
+
+  return (
+    <EuiForm component={'form'} id={'startDeploymentForm'}>
+      <EuiDescribedFormGroup
+        title={
+          <h3>
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.startDeployment.numbersOfAllocationsLabel"
+              defaultMessage="Number of allocations"
+            />
+          </h3>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.ml.trainedModels.modelsList.startDeployment.numbersOfAllocationsHelp"
+            defaultMessage="Increasing the number of allocations will increase throughput, providing your ML nodes have sufficient processors."
+          />
+        }
+      >
+        <EuiFormRow
+          label={
```

Comment:
I'm not sure this label is needed as it duplicates the text in the `EuiDescribedFormGroup` title.

Reply:
I assume it's needed to accessibility